recipe/meta.yaml

```diff
@@ -36,13 +36,22 @@ requirements:
 test:
   requires:
     - pip
+    # Running unit tests requires bottle.
+#    - pytest >=5
+#    - bottle
```

Comment:
You can try this [patch](https://github.com/pycurl/pycurl/commit/79389aef7f41c8b1eaef00766602c0ac1f90dcf8.patch) and `flask` to unblock the test suite